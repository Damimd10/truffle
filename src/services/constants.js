export const URL =
  'http://tm-kitchen-api-alpha.herokuapp.com/videos?api_key=homework&auth_token=1&limit=10&workflow_status=ready';
