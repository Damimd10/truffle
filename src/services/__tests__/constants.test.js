import { URL } from '../constants';

describe('Constants File', () => {
  it('should match with the url of the snapshot', () => expect(URL).toMatchSnapshot());
});
