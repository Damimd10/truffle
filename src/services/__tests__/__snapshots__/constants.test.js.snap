// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Constants File should match with the url of the snapshot 1`] = `"https://tm-kitchen-api-alpha.herokuapp.com/videos?api_key=homework&auth_token=1&limit=10&workflow_status=ready"`;
